README.md

```diff
@@ -41,7 +41,22 @@ in your commit messages so it is clear what each commit is for._" <br />
 when you want someone to review your work..._" <br />
 > "_Please only assign a Pull Request for review
 once all the tests have passed on CI..._" <br />
-> For a **list** of these common scenarios see:
+
+We've devised a set of rules and actions to respond to these problems, they look like this:
+
+| Category      | Rule                                                                               | dwylbot comments                                                                                                                                                                                                                                                                                                      | Labels changed                                                                   |
+|---------------|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
+| Pull Requests | "awaiting-review" and a merge conflict, #64                                        | :warning: @username, the pull request has a **merge conflict**., Please resolve the conflict and reassign when ready :+1:, Thanks!                                                                                                                                                                                    | remove "awaiting-review" and replace with "merge-conflicts". @username assigned. |
```

Comment:
The issue number ie: #64 are not linked to the issues, we might need to precise the full url?